src/clj/puppetlabs/services/ca/certificate_authority_core.clj

```diff
@@ -27,6 +27,12 @@
 ;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
 ;;; 'handler' functions for HTTP endpoints
 
+(defn handle-server-error
+  ([message]
+    (-> (rr/response message)
+        (rr/status 400)
+        (rr/content-type "text/plain"))))
+
```

Comment:
@KevinCorcoran it's probably as if there were multiple signatures for the function, except only one is defined